typings/index.d.ts

```diff
@@ -3294,6 +3302,8 @@ declare module 'discord.js' {
     type?: OverwriteType;
   }
 
+  type NsfwLevel = 'DEFAULT' | 'EXPLICIT' | 'SAFE' | 'AGE_RESTRICTED';
```

Comment:
```suggestion
  type NsfwLevel = keyof typeof NsfwLevels;
```
This should also be moved above `OverwriteData`

Reply:
```suggestion
  type NSFWLevel = keyof typeof NSFWLevels;
```